<launch>
    
    <!-- 机器人模型 和 rviz显示 -->
	<param name="robot_description" command="$(find xacro)/xacro $(find hit_spider)/model/xacro/EISpider.xacro" />
    <node pkg="robot_state_publisher" type="robot_state_publisher" name="robot_state_publisher" output="screen" />
    <!-- 显示程序 -->
    <node pkg="hit_spider" type="walkingNode" name="walkingNode" output="screen" />
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find hit_spider)/config/walking_rviz.rviz" output="screen" />

    <!-- 静态坐标变换 -->
    <node pkg="hit_spider" type="static_information_publisher" name="static_information_publisher" output="screen" />
    <!-- MCTS规划程序 -->
    <!-- <node pkg="hit_spider" type="main" name="main_node" output="screen" /> -->

</launch>